pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -76,7 +76,7 @@
       <Version>1.0.2</Version>
     </NETCoreApp10Dependency>
     <NETCoreApp10Dependency Include="Microsoft.NETCore.Runtime.CoreCLR">
-      <Version>1.0.12-servicing-26117-01</Version>
```

Comment:
You need to update DotNetHostPolicy here as well to the 1.0.0 equivalent. I think it’s 1.0.11?

Reply:
Wouldn't it be 1.0.10? https://github.com/dotnet/core-setup/pull/3664/files#diff-c0a524ec6ca51973079c79a8e15af029R324

Reply:
Yes. 1.0.10